src/lib/expansion/expansion-panel-header.ts

```diff
@@ -86,17 +88,24 @@ export class MatExpansionPanelHeader implements OnDestroy {
 
   constructor(
     renderer: Renderer2,
+    @Optional() accordion: MatAccordion,
     @Host() public panel: MatExpansionPanel,
     private _element: ElementRef,
     private _focusMonitor: FocusMonitor,
     private _changeDetectorRef: ChangeDetectorRef) {
 
+    let changeSubjects = [panel._inputChanges];
```

Comment:
You don't care here that these are Subjects, just that they're observables. I'd call this `changeStreams`